versioned/mongodb/scripts/cfg.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I'd like to see if we can do this without any scripts. Can you please explore some alternatives?

Reply:
This has been superseded. We now how sample scripts and a README.md.